docs/installing.rst

```diff
@@ -116,10 +124,10 @@ If you want to have access to the sources you can run the installation in a slig
 
   git clone https://github.com/ICRAR/daliuge
 
-Then perform the following steps to setup and install |daliuge| into an isolated environment::
+Perform the following steps to setup and install |daliuge| into a virtual environment::
 
   cd daliuge
-  make virtualenv
+  python -m venv .venv
   source .venv/bin/activate
   make show # Optional, use to confirm virtualenv is active
   make install
```

Comment:
This should be changes to `make local`, this was an error when I originally updated the docs.